frontend/__tests__/services/wsaddress-service.server.test.ts

```diff
@@ -0,0 +1,34 @@
+/* eslint @typescript-eslint/no-unused-vars: ["error", { "varsIgnorePattern": "^_" }] */
+import { describe, expect, it } from 'vitest';
+import { ParseWSAddressResponseDTO } from '~/dtos/parse-address-response-dto.server';
+import { ValidateWSAddressResponseDTO } from '~/dtos/validate-address-response-dto.server';
+import { server } from '~/mocks/node';
+import { getWSAddressService } from '~/services/wsaddress-service.server';
+
+const WSAddressService = await getWSAddressService();
+
+vi.mock('~/utils/env.server', () => ({
+  getEnv: vi.fn().mockReturnValue({ INTEROP_API_BASE_URI: 'https://api.example.com' }),
+}));
+
+describe('wsaddress-service.server tests', () => {
+  beforeAll(() => server.listen({ onUnhandledRequest: 'bypass' }));
+  afterEach(() => {
+    vi.clearAllMocks();
+    vi.resetModules();
```

Comment:
**Nitpick:** I don't think `vi.resetModules()` is required here since none of your imports have side effects. (I could be wrong about that one)